backend/open_webui/utils/chat.py

```
@@ -7,14 +7,17 @@
 import random
 import json
 import inspect
+import uuid
+import asyncio
 
-from fastapi import Request
-from starlette.responses import Response, StreamingResponse
+from fastapi import Request, status
```

Ruff (0.8.2)

`fastapi.status` imported but unused

Remove unused import: `fastapi.status`

(F401)